pytype/pyc/opcodes_test.py

```diff
@@ -28,6 +29,14 @@ def assertDisassembly(self, code, expected):
       else:
         self.assertEqual(e, (o.name, o.arg))
 
+  def assertLineNumbers(self, code, co_lnotab, expected):
+    """Assert that the opcodes in a code sequence have the expected
```

Comment:
see if you can fit this on one line; if not, the linter requires multiline doc strings to have complete documentation of args. also needs a `.` at the end of the first line.